src/current/_includes/releases/cloud/2024-05-20.md

```diff
@@ -0,0 +1,5 @@
+## May 20, 2024
+
+<h3 id="2024-05-20-security-updates"> Security updates </h3>
+
+- [Configuring private connectivity using Google Cloud Private Service Connect]({% link cockroachcloud/connect-to-your-cluster.md %}#gcp-private-service-connect) is available in [preview](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroachdb-feature-availability) for CockroachDB {{ site.data.products.dedicated }} clusters on GCP. [Private connectivity]({% link cockroachcloud/network-authorization.md %}#options-for-controlling-network-access) allows you to establish SQL access to a CockroachDB {{ site.data.products.dedicated }} cluster entirely through cloud provider private infrastructure, without exposing the cluster to the public internet, affording enhanced security and performance.
```

Comment:
@devarshishah3 PTAL, this is modeled on the note we used for AWS PrivateLink on multi-region Serverless.